src/GregClientSandbox/Program.cs

```diff
@@ -21,8 +22,9 @@ internal class Samples
         /// A GregClient specifying the local host. The alternate IP provided is for 
         /// production Reach. 
         /// </summary>
-        //private static GregClient pmc = new GregClient(provider, "http://107.20.146.184/");
-        private static GregClient pmc = new GregClient(provider, "http://localhost:8080/");
+        private static GregClient pmc = new GregClient(provider, "http://107.20.146.184/");
+        //private static GregClient pmc = new GregClient(provider, "http://localhost:8080/");
+        //private static GregClient pmc = new GregClient(provider, "http://dynamopackages.com/");
```

Comment:
@mjkkirschner The other tests will continue to "pass" (they're not really unit tests), because their routes exist on the package manager service.